src/libstd/panicking.rs

```diff
@@ -278,36 +278,36 @@ pub unsafe fn r#try<R, F: FnOnce() -> R>(f: F) -> Result<R, Box<dyn Any + Send>>
         Err(ManuallyDrop::into_inner(data.p))
     };
 
-    // Compatibility wrapper around the try intrinsic for bootstrap
-    #[inline]
+    // Compatibility wrapper around the try intrinsic for bootstrap.
+    //
+    // We also need to mark it #[inline(never)] to work around a bug on MinGW
+    // targets: the unwinding implementation was relying on UB, but this only
```

Comment:
Is there at least a brief explanation somewhere of what the UB is that it was relying on? I could not find that here nor in https://github.com/rust-lang/rust/issues/70001.

Reply:
https://github.com/rust-lang/rust/pull/67502#issuecomment-586584698
https://github.com/rust-lang/rust/pull/67502#issuecomment-586588392

Reply:
Basically we were replacing LLVM's `resume` instruction with calls to `rust_eh_unwind_resume` (which then forwards to `_Unwind_Resume`), but it turns out that LLVM actually needs to make certain code transformations on landing pads which weren't being done since we were not using the proper `resume` instructions.

Reply:
Thanks!